server.js

```diff
@@ -43,7 +44,8 @@ function getData(event) {
     return new Promise ((resolve, reject) => {
         requests.listEvents(oauth2Client)
             .then(requestEvents => {
-                resolve(requestEvents);
+                let filteredEvents = filtering.formatEvents(requestEvents, event.id);
```

Comment:
`filtering.formatEvents` is a bit confusing, it's either filtering or formatting :)
It would be nice to have something like format(requestEvents.filter(`filteringFunction here`))